src/main/groovy/edu/oregonstate/mist/inventory/resources/InventoryResource.groovy

```diff
@@ -62,15 +57,15 @@ class InventoryResource extends Resource {
     @GET
     @Path('{id: [0-9a-zA-Z-]+}')
     Response getInventoryByID(@PathParam("id") String inventoryID) {
-        Inventory inventory = inventoryDAO.getInventoryByID(inventoryID)
+        ResourceObject inventory = inventoryDAOWrapper.getInventoryById(
+                inventoryID, endpointUri
```

Comment:
Does the endpointUri change per request? I'm wondering if the endpiontUri should be something that you pass to the constructor of iventoryDAOWrapper.